.gitlab-ci.yml

```diff
@@ -25,10 +25,12 @@ include:
   - project: 'Northern.tech/Mender/mendertesting'
     file: '.gitlab-ci-github-status-updates.yml'
 
-test:unit:
-  image: golang:1.14
-
 test:acceptance_tests:
   variables:
     DOCKER_HOST: tcp://docker:2375/
     DOCKER_DRIVER: overlay2
+  script:
+    # Remove coverage from previous runs (if any)
+    - rm -f tests/coverage-acceptance*.txt
```

Comment:
Why not adding this to [the template](https://github.com/mendersoftware/mendertesting/blob/master/.gitlab-ci-check-docker-acceptance.yml#L101) instead? It is a harmless command.

Reply:
The script section in the template will not work since it expects a different coverage file for the enterprise run. Since I'm completely overriding the script I added this line in particular to make the job reentrant.

Reply:
Ok, got it. But if we move towards this approach in all repos is best to have the template updated to handle this case (same coverage file for enterprise run).